netspeed/data/org.mate.panel.applet.netspeed.gschema.xml.in

```diff
@@ -16,7 +16,7 @@
       <description>If true, show speed in bits instead of bytes.</description>
     </key>
     <key name="show-all-addresses" type="b">
-      <default>false</default>
+      <default>true</default>
```

Comment:
I don't think it should be enabled by default

Reply:
I think it should be. IPv6 is here and it is here to stay. It's time for people to recognize it. Whoever doesn't like it can go and turn it off.